### PR TITLE
Added Digit Count program

### PR DESCRIPTION
**Pull request by Utsav1999**
# Welcome to JavaScript community

### **Describe your change:**
I added a program that will ask a user to give a number as an input. Then it will count the number of digits present in the number and will print it to the console.

### CODE REVIEW

**Comment by Utsav1999** [comment]
how to add npm standards?

**Comment by nandanvasudevan** [comment]
> how to add npm standards?

Just run npm install, it will install all the dependecies.
Then run `standard CountDigit.js`
***Fix the errors and push***

**Comment by Sheraff** [comment]
This doesn't work for `0` I think. It will return `-Infinity`.

**Comment by Utsav1999** [comment]
@Sheraff thanks for notifying me. I have updated my program and it's working fine now.

File: Maths/CountDigit.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 */
 
 function CountDigit (number) {
-    var digits = Math.floor(Math.log10(number) + 1);
-    return digits;
+  var digits = Math.floor(Math.log10(number) + 1)
+  return digits
 }
 
-console.log('The number of digits in the number: ' + CountDigit(parseInt(prompt('Enter number: '))));
```
Comment (itsvinayak) [inline Maths/CountDigit.js]: @Utsav1999, thank you for your contribution 

this is from the library function
`
const prompt = require('prompt-sync')();`
this line is required
we are using nodejs to run and test code.
prompt is a lib function to take input, _window.prompt_ will not work in node
**to install**
`npm install prompt-sync`

[resource](https://www.codecademy.com/articles/getting-user-input-in-node-js#:~:text=Once%20you've%20loaded%20the,Hey%20there%20%24%7Bname%7D%60) 

@itsvinayak

**Comment by Utsav1999** [comment]
any update @itsvinayak ?

**Pull request closed without merging**